### DISCUSSION TITLE
about terminal

### DISCUSSION THREAD

**Discussion by AdnanAhad** [community/community#6991] [Codespaces] [answered]
when terminal is coming in vscode.dev

**Comment by bamurtaugh** [comment, marked as answer]
Hi @AdnanAhad, VS Code for the Web (vscode.dev) and the customized instance of it at github.dev provide a lightweight version of VS Code running entirely in the browser. Since VS Code for the Web is running completely within the browser, some experiences will naturally be more constrained, when compared to what you can do in the VS Code desktop app or a Codespace. 

Also, to ensure your future feedback is routed appropriately - this forum (https://github.com/github/feedback/discussions) is designed for Codespaces-specific issues, including GitHub Codespaces and the github.dev web editor. Issues on VS Code for the Web (vscode.dev) can be filed in the [microsoft/vscode](http://github.com/microsoft/vscode) repo. Thank you!

**Reply by AdnanAhad** [reply]
I can only say that you should add terminal to it...thanks
